### ISSUE TITLE
Ability to load additional idris packages

### ISSUE THREAD

**Issue by archaeron** [meraymond2/idris-ide-client#1]
Hi

thanks for your work on this. This looks great and I'm trying to integrate it into the atom plugin.

I hope it's okay if I open some issues on things that would be nice to have.
I'm happy to help resolve them.

**Comment by archaeron** [comment]
I need this for the atom plugin. I'll try to do something as a package that we can both reuse like you did :)
I'll try to keep it simple.

**Comment by archaeron** [comment]
I changed the title because this project only needs the ability to load additional idris packages from the `libdir` (`idris --libdir`)

Pull request incoming

**Comment by archaeron** [comment]
this won't be needed after #7
